backend/corpora/parliament/tests/test_import.py

```diff
@@ -3,47 +3,49 @@
 import pytest
 from datetime import datetime
 
+from addcorpus.python_corpora.corpus import CorpusDefinition
 from addcorpus.python_corpora.load_corpus import load_corpus_definition
 
 CORPUS_TEST_DATA = [
```

Comment:
It takes a lot of scrolling to get to the actual tests in this file. I propose making `CORPUS_TEST_DATA` a fixture in a separate file.